users/views_api.py

```diff
@@ -0,0 +1,60 @@
+import django.contrib.auth.password_validation as password_validation
+from django.contrib.auth.models import User
+from rest_framework import generics, views
+from rest_framework.authtoken.models import Token
+from . import serializers
+from . import permissions
+
+
+class UserListCreateView(generics.ListCreateAPIView):
```

Comment:
At the moment we don't need an endpoint to list all users in the database. Let's remove the listing user functionality.